src/liquid/components/ld-radio/ld-radio.tsx

```diff
@@ -227,7 +227,7 @@ export class LdRadio implements InnerFocusable {
           onKeyDown={this.handleKeyDown}
           ref={(ref) => (this.input = ref)}
           type="radio"
-          {...cloneAttributes(this.el, ['autocomplete', 'tone', 'mode'])}
+          {...cloneAttributes.call(this, ['autocomplete', 'tone', 'mode'])}
```

Comment:
```suggestion
          {...cloneAttributes.call(this, ['tone', 'mode'])}
```

Reply:
This is not related to the PR. But 🤔 Should I also remove it in `ld-checkbox` and `ld-toggle`?

Reply:
Well, would probably make sense. Maybe you can add it to this PR?